docs/MTX.md

```diff
@@ -54,11 +57,11 @@ Note: 'x' refers to the serial port of your flight controller used for MAVLink
 
 ### Stream Rates
 
-In contrast to the other setups, the SRy parameters for the stream rates do not need to be configured here (i.e., can be left at zero). The MAVLink for OpenTx firmware requests the required streams itself.
+In contrast to other setups, the SRy parameters for the stream rates do not need to be configured here (i.e., can be left at zero). The mTX firmware requests the required streams itself.
```

Comment:
On CRSF page we recommend SERIAL_OPTIONS = 4096 | ignore stream rate change.  Should line be added here that this bit should not be set in the bitmask?

Reply:
yeah ... I would wish ArduPilot would support more options here.

Reply:
hm, thinking about it, I'm actually not sure how true the sentence is, i.e., if it indeed has absolutely no effect. Maybe it has some side effects. I guess one should say on can try ...